SW2URDF/SW/SwAddin.cs

```diff
@@ -366,7 +366,6 @@ public void SetupPartExporter()
                 PartExportForm exportForm = new PartExportForm((SldWorks)SwApp);
                 logger.Info("Showing part");
                 exportForm.Show();
-                exportForm.Dispose();
```

Comment:
This form show and then dispose. So window appears and then quickly disappears quickly.